templates/base/package.json

```diff
@@ -43,14 +43,14 @@
   "devDependencies": {
     "@next/bundle-analyzer": "^9.3.6",
     "@babel/plugin-transform-runtime": "^7.9.6",
-    "@testing-library/jest-dom": "^5.5.0",
-    "@testing-library/react": "^10.0.4",
-    "@testing-library/react-hooks": "^3.2.1",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^12.1.0",
+    "@testing-library/react-hooks": "^7.0.2",
     "babel-plugin-transform-flow-strip-types": "^6.22.0",
     "bundlesize": "^0.18.0",
     "cross-env": "^7.0.2",
-    "jest": "25.2.7",
-    "jest-mock-console": "^1.0.0",
+    "jest": "27.2.2",
+    "jest-mock-console": "^1.1.0",
```

Comment:
Have you validated with the config file, is it fine & supporting all the existing configs?

- check for test runners
- environments

as these are most likely affected, either they are included in the base setup by default?

Two things which we can discuss further is

- jest-mock-console
- @testing-library/react-hooks

do these require to be included in the base setup?